Civi/API/Kernel.php

```diff
@@ -157,6 +157,20 @@ public function runRequest($apiRequest) {
     catch (\Exception $e) {
```

Comment:
Maybe do an earlier catch for the exception rather than testing for it within the broad-catch